src/main/java/org/folio/circulation/domain/validation/ItemStatusValidator.java

```diff
@@ -36,9 +36,18 @@ public ItemStatusValidator(
       });
   }
 
-  public Result<LoanAndRelatedRecords> refuseWhenItemStatusIsInvalid(Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
+  public Result<LoanAndRelatedRecords> refuseWhenItemIsMissing(
+    Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
+
     return loanAndRelatedRecords
       .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.MISSING))
       .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.DECLARED_LOST));
   }
+
+  public Result<LoanAndRelatedRecords> refuseItemWithDeniedStatuses(
+    Result<LoanAndRelatedRecords> loanAndRelatedRecords) {
+
+    return refuseWhenItemIsMissing(loanAndRelatedRecords)
+      .next(p -> checkAndRefuseItem(loanAndRelatedRecords, ItemStatus.CLAIMED_RETURNED));
+  }
```

Comment:
I like the idea of combining this into one method. I wonder if there is a better name for the collective set of statuses than denied.

I think it could be more readable by inlining the `refuseWhenItemIsMissing` method, as the three lines are then presented together and similar.

I might also statically import the `ItemStatus` values to reduce the noise.

Reply:
Maybe `refuseWhenItemIsNotAllowedForCheckOut` or `refuseWhenItemIsNotPresentInInventory`?

Reply:
I'm happy with `refuseWhenItemIsNotAllowedForCheckOut`